fake_svc/otel-collector-config.yaml

```diff
@@ -0,0 +1,46 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+  # Receiver for reading logs from stdout
+  filelog:
+    include:
+      - /var/log/otel-logs/*.log  # Log files being written to stdout or any specific log file
+    start_at: beginning
+
+exporters:
+  otlp:
+    endpoint: "jaeger:4317"  # Point to Jaeger's OTLP gRPC endpoint
+    tls:
+      insecure: true
+
+  otlphttp:
+    endpoint: http://localhost:3100/otlp  # Endpoint for Loki or other log receivers
+
+processors:
+  batch:
+    send_batch_size: 1024
+    timeout: 10s
+  attributes:
+    actions:
+      - action: insert
+        key: service.name
+        value: fake-service
```

Comment:
what's this?

Reply:
this will add an attribute, i.e. label to the logs which will be `service_name=fake-service` so we can query better
see https://grafana.com/docs/loki/latest/configure/#attributes_config